cloud/roles/postgres_connection/tasks/create-databases.yml

```diff
@@ -0,0 +1,42 @@
+- name: Create the "{{ vars[ cluster_type + '_database_user'] }}" role for manage "{{ vars[ cluster_type + '_database_name'] }}" database
+  postgresql_user:
+    login_host:      "{{ control_database_host }}"
+    login_password:  "{{ control_db_admin_password | b64decode }}"
+    name:            "{{ vars[ cluster_type + '_database_user'] }}"
+    password:        "{{ vars[ cluster_type + '_db_user_password'] | b64decode }}"
+    role_attr_flags: CREATEROLE
+
+- name: Create the "{{ vars[ cluster_type + '_database_name'] }}" database
+  postgresql_db:
+    login_host:      "{{ control_database_host }}"
+    login_password:  "{{ control_db_admin_password | b64decode }}"
+    name:            "{{ vars[ cluster_type + '_database_name'] }}"
+    owner:           postgres
+
+- name: Add privileges for the "{{ vars[ cluster_type + '_database_user'] }}" role to the "{{ vars[ cluster_type + '_database_name'] }}" database
+  postgresql_privs:
+    login_host:      "{{ control_database_host }}"
+    login_password:  "{{ control_db_admin_password | b64decode }}"
+    db:              "{{ vars[ cluster_type + '_database_name'] }}"
+    role:            "{{ vars[ cluster_type + '_database_user'] }}"
+    type:            database
+    objs:            "{{ vars[ cluster_type + '_database_name'] }}"
+    privs:           ALL
+
+- name: Add postgres to membership of the "{{ vars[ cluster_type + '_database_user'] }}" role
```

Comment:
I don't get it. Can you rephrase? "add '_database_user' role postgres"?

Reply:
Why don't you understand it? https://www.postgresql.org/docs/9.0/static/role-membership.html. Maybe change to member of role ? This phrase `"grant '_database_user' role postgres"` isn't understand for me.

Reply:
add postgres to role '_database_user` ?

Reply:
Done.